mailpoet/assets/js/src/automation/editor/components/workflow/types.ts

```diff
@@ -1,3 +1,5 @@
+import { WorkflowStatus } from '../../../listing/workflow';
```

Comment:
Hmm, not urgent, but we should probably move some of these types to the top-level components directory so editor doesn't need to know about listings and vice-versa.

Reply:
Added an issue [MAILPOET-4748] to handle this.